docs/DeveloperGuide.md

```diff
@@ -624,202 +625,210 @@ testers are expected to do more *exploratory* testing.
-
-      E.g. `shutter` from the name field will be matched.
+   4. Test case: Invalid command format
+      1. Command: `find n/` <br>
+         Expected: No show is found. Error details shown in the result display, with a result message saying <br> `Invalid command format!...`
+      2. Command: `find t/Action Anime` <br>
+         Expected: No show is found. Error details shown in the result display, with a result message saying <br> `Invalid command format!...`
    
-   9. Test case: `find t/Action Anime`
+   5. Test case: Multiple same prefix
+      1. Command: `find n/shingeki n/shutter` <br>
+         Expected: Looks through the name field for any partial or full word of `shutter` then displays them on the show list. (Ignores the first instance of n/) <br>
+         E.g. `shutter` from the name field will be matched.
 
-      Expected: No show is found. Error details shown in the result display, with a result message saying `Invalid command format!...`
-   
 [return to top <img src="images/toc-icon.png" width="25px">](#table-of-contents)
 
 ---
 
 ### Sorting the list of shows
 
-1. Sorting the list of shows
+1. Prerequisites: None, but if the list is empty, all sorts will lead to no results.
+2. Test case: Sort with no prefix
+   1. Command: `sort` <br>
+      Expected: The list of shows will default to being sorted by name in ascending order.
 
-    1. Prerequisites: None, but if the list is empty, all sort will also lead to no results.
-
-    2. Test case: `sort`<br>
-       Expected: The list of show sorted by name in ascending order.
-
-    3. Test case: `sort sna/`<br>
-       Expected:  The list of show sorted by name in ascending order.
-   
-    4. Test case: `sort snd/`<br>
-       Expected:  The list of show sorted by name in descending order.
+3. Test case: Sort with single prefix 
+   1. Command: `sort sna/` <br>
+      Expected: The list of shows is sorted by name in ascending order.
    
-    5. Test case: `sort ssa/`<br>
-       Expected:  The list of show sorted by status in ascending order.
-   
-    6. Test case: `sort ssd/`<br>
-       Expected:  The list of show sorted by status in descending order.
+   2. Command: `sort snd/` <br>
+      Expected: The list of shows is sorted by name in descending order.
+
+   3. Command: `sort ssa/` <br>
+      Expected: The list of shows is sorted by status in ascending order.
    
-    7. Test case: `sort sna/ snd/`<br>
-       Expected: The list of show sorted by name in ascending order.
+   4. Command: `sort ssd/` <br>
+      Expected: The list of shows is sorted by status in descending order.
    
-    8. Test case: `sort ssa/ ssd/`<br>
-       Expected: The list of show sorted by status in ascending order. 
+4. Test case: Sort with multiple same prefixes 
+   1. Command: `sort sna/ snd/` <br>
+      Expected: The list of show sorted by name in ascending order.
```

Comment:
Should this be `list of shows` for consistency?

Reply:
Edited